webapp/src/domain/runs/JsonPathSearchToolbar.tsx

```diff
@@ -137,15 +137,15 @@ export default function JsonPathSearchToolbar(props: ToolbarProps) {
                                 </DropdownToggle>
                             }
                             dropdownItems={[
-                                <DropdownItem id="js" key="query">
-                                    js
-                                </DropdownItem>,
                                 <DropdownItem id="jsonb_path_query_first" key="jsonb_path_query_first">
                                     jsonb_path_query_first
                                 </DropdownItem>,
                                 <DropdownItem id="jsonb_path_query_array" key="jsonb_path_query_array">
                                     jsonb_path_query_array
                                 </DropdownItem>,
+                                <DropdownItem id="js" key="query">
+                                    js
+                                </DropdownItem>
```

Comment:
```suggestion
                                <DropdownItem id="js" key="query">
                                    js
                                </DropdownItem>
```

Reply:
@whitingjr why we are showing only js text?

Reply:
@shivam-sharma7  What I've done is relocate these lines of code. When you look at the context the [dropdownItems](https://github.com/whitingjr/Horreum/blob/9c1656608f068fbac27784aeb406be77d9392f4f/webapp/src/domain/runs/JsonPathSearchToolbar.tsx#L139-L148) still has 3 items.